automation/test.sh

```diff
@@ -2,7 +2,13 @@
 
 main() {
     ls -l
+    echo "Installing promtool..."
+    VERSION="2.42.0"
```

Comment:
Shouldn't we fetch the latest version [1] instead of having a fixed version.
Otherwise this will have to be updated manually. 

[1] https://github.com/prometheus/prometheus/releases/latest

Reply:
The original script I used as source had it as latest. I thought we might prefer having it fixed to avoid surprises.

Reply:
@yftacherzog Having it as a fixed version often makes more sense if people remember to bump the version manually. From past experience (for example with the Pipfile we had with some of the dependencies having fixed versions), people don't bump the version and then we end up with a very outdated version.
Anyway, I'm fine with both approaches (personally, I prefer yours better to avoid unexpected regressions).

Reply:
I think it's fine either way for now. Personally, I also tend to prefer fixed versions for the same reason @eisraeli stated.